container-storage-setup.1

```diff
@@ -88,8 +88,11 @@ functionality to work properly.
 STORAGE_DRIVER:
       Specify a storage driver to be used with container runtime.
       Default: "devicemapper".
-      Valid values are overlay, overlay2 and "".
-      "" tells container-storage-setup to not perform any storage setup.
+      Valid values are overlay, try-overlay, overlay2 and "".  The
```

Comment:
How about introducing a new option instead. Say `STORAGE_DRIVER_AUTO=1`. If this is set, then storage driver is determined automatically by the script. And it could be anything. I mean down the line we might want to configure a new driver say "xfs reflink" first and then fall back to overlay and then fall back to devicemapper. This option will continue to work even when we introduce new drivers.

Reply:
STORAGE_DRIVER_AUTO basically means determine storage driver automatically without user having to specify a fixed value.

Reply:
How about `s/try-overlay/auto/`?

Reply:
STORAGE_DRIVER=auto sounds good.